sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -32,39 +32,37 @@ stripe_customers AS (
   FROM
     mozdata.stripe.customers
 ),
-fxa_logs_provider_country AS (
+paypal_country AS (
   SELECT
     fxa_uid,
     plan_id,
-    ARRAY_AGG(
-      STRUCT(payment_provider AS provider, source_country AS country)
-      ORDER BY
-        event_timestamp DESC
-      LIMIT
-        1
-    )[OFFSET(0)].*,
+    ARRAY_AGG(source_country ORDER BY event_timestamp DESC LIMIT 1)[OFFSET(0)] AS country,
   FROM
     `moz-fx-data-shared-prod`.stripe_external.fxa_pay_setup_complete_v1
+  WHERE
+    LOWER(payment_provider) = "paypal"
   GROUP BY
     fxa_uid,
     plan_id
 ),
+stripe_charges AS (
+  SELECT
+    id AS charge_id,
+    payment_method_details.card.country,
+  FROM
+    `moz-fx-data-shared-prod`.stripe_external.charges_v1
+  WHERE
+    status = "succeeded"
```

Comment:
moved this to a common table expression so that status can be filtered before the join